tawk_to.module

```diff
@@ -310,7 +306,7 @@ function tawkto_widget_options()
             }
         }
     }
-    variable_set(TAWKTO_WIDGET_OPTIONS, json_encode($jsonOpts));
+    config_set(TAWK_TO_CONFIG_NAME, TAWK_TO_WIDGET_OPTIONS, json_encode($jsonOpts));
```

Comment:
@yorkshire-pudding can we also add the cache flushing here? 😁

Reply:
Done